Rmarkdown/Multiple_Ascending_Dose_PD_real_example.Rmd

```diff
@@ -166,49 +155,56 @@ plot2
 For this fast acting drug, an effect in change from baseline intensity score can actually be seen within the first 24 hours. In order to get an idea of the dose response relationship and make sure we are targeting an optimal dose, take a look at the response for a range of doses that were studied in the single ascending dose study. Plotting boxplots of the change from baseline intensity score against dose, you can see that starting at 10 mg, drug ABC123 has a clear effect on intensity score. 
 
 
-```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+```{r, cache = TRUE, fig.width = 10, fig.height = 4, warning = FALSE, message = FALSE}
  
 labelFun <- function(variable,value){     
   paste0(value," h")                             
 }
 
-ytitle<-ylab(paste0(cmt_name," ",units,"\n change from baseline"))
-xtitle<-xlab("Dose (mg)")
-
-plot1<-ggplot(data_to_plot_single_dosing, aes(y=LIDV2, x= factor(DOSE), group=NTIM)) + 
-  geom_boxplot(aes(group = factor(DOSE)))+
-  ggtitle("Single dosing")+ 
-  theme_bw() +
-  ytitle+
-  xtitle+
-  geom_hline(yintercept=0,color="red",linetype="dashed")+
-  theme(axis.text=element_text(size=13),axis.title=element_text(size=14),strip.text=element_text(size=14))+
-  facet_wrap(~NTIM,nrow=1,label=labelFun)
-
-plot1 
+ytitle <- ylab(paste0(pd_label,"\nchange from baseline"))
+xtitle <- xlab("Dose (mg)")
+
+gg <- ggplot(pd_single_dosing, 
+                aes(y = CHG, x = factor(DOSE), group = NOMTIME))  
+gg <- gg + geom_boxplot(aes(group = factor(DOSE)))  
+gg <- gg + ggtitle("Single dosing")   
+gg <- gg + ytitle + xtitle   
+gg <- gg + geom_hline(yintercept = 0,
+             color = "red",linetype = "dashed")   
+gg <- gg + theme(axis.text  = element_text(size = 13),
+                 axis.title = element_text(size = 14),
+                 strip.text = element_text(size = 14))   
+gg <- gg + facet_wrap(~NOMTIME, nrow = 1, label = labelFun)
```

Comment:
Are you sure you want to create both plots?  I wasn't totally sure what the purpose was of the left-hand plot.  But if you have a reason for wanting it, I'm ok to keep it.

Reply:
removed